schedule/models/events.py

```diff
@@ -136,7 +225,7 @@ def _get_occurrence_list(self, start, end):
             if self.end_recurring_period and self.end_recurring_period < end:
                 end = self.end_recurring_period
 
-            rule = self.get_rrule_object()
+            rule = self.get_optimized_rrule_object(start-difference)
```

Comment:
ok, sure. Will add that. We use the "ENABLE_" convention elsewhere ... so should I stick with that. So call it 'ENABLE_OPTIMIZED_RRULE_GENERATION'. Also, get_boolean() is defined in vishnu-backend and we likely don't want dependencies going from scheduler up to there. So I'd probably have to recreate that in this repo as well.